drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -708,6 +690,7 @@ function DragAndDropBlock(runtime, element, configuration) {
                         releaseItem($selectedItem);
                     } else if (isActionKey(evt)) {
                         evt.preventDefault();
+                        evt.stopPropagation();
```

Comment:
@e-kolpakov Thanks for the explanation, I had wondered about that briefly but somehow forgot to comment about it - you read my mind :)